shared/uavobjectdefinition/charonscreendisplaysettings.xml

```diff
@@ -2,6 +2,16 @@
 	<object name="CharOnScreenDisplaySettings" singleinstance="true" settings="true">
 		<description>Character On Screen Display Settings</description>
 
+		<field name="VideoStandard" units="enum" type="enum" elements="1" defaultvalue="AutodetectPreferPAL">
+			<description>The video standard to use for the OSD</description>
+			<options>
+				<option>AutodetectPreferPAL</option>
```

Comment:
You can use spaces (+some other punctuation) in this to look nicer in UI, and it will be collapsed in the generated UAVO code. :stuck_out_tongue: